### PR TITLE
Add reviews styling

### PR DESCRIPTION
**Pull request by JimmyHoenderdaal**

### CODE REVIEW

File: resources/views/reviews.blade.php [hunk 1]
```diff
@@ -24,4 +23,20 @@
             </div>
         </graphql>
     </lazy>
+    @else
```
Comment (royduin) [inline resources/views/reviews.blade.php:26]:
```suggestion
@else
```

File: resources/views/reviews.blade.php [hunk 2]
```diff
@@ -24,4 +23,20 @@
...
+    <div class="w-full p-8 mb-4 border rounded bg-white">
+        <div class="flex items-center gap-2 text-secondary">
+            <strong itemprop="name">@lang('No reviews found')</strong>
+            <span class="text-xs">
+                @{{ new Date(Date.now()).toLocaleDateString() }}
```
Comment (royduin) [inline resources/views/reviews.blade.php:31]:
```suggestion
                {{ Carbon::now()->toDateString() }}
```
But why is the date displayed here?

File: resources/views/form.blade.php [hunk 3]
```diff
@@ -33,3 +33,31 @@
         </div>
     </graphql>
 </lazy>
+
+{{-- TODO: Fix the above code and remove this static HTML --}}
```
Comment (royduin) [inline resources/views/form.blade.php:37]: So what's the problem?

**Review by royduin — CHANGES_REQUESTED** [review]

File: resources/views/reviews.blade.php (hunk 2 above)
Comment (JimmyHoenderdaal) [reply resources/views/reviews.blade.php:31]: As a placeholder, it looks nice

File: resources/views/form.blade.php (hunk 3 above)
Comment (JimmyHoenderdaal) [reply resources/views/form.blade.php:37]: Not sure @jbclaudio said he would like to look into it

**Pull request closed without merging**